[PR-17968] Update documentation for loyalty and email capture integration (#268)
talkable/talkable-docs: source/integration/loyalty/integration_components.rst

    - Email
    - Phone Number
    - First Name
    - Last Name
    - Custom Properties

2. :ref:`Talkable Container DIV`

>    **DIV Location.** Talkable Container DIV script should be placed in the body of every page where you want to show a loyalty campaign. This DIV determines the place where the campaign is shown.

Reviewer: ```suggestion
   **DIV Location.** Talkable Container DIV should be placed in the body of every page where you want a loyalty campaign to be shown. This DIV tag determines the place where the campaign is shown.
```
And please provide example of the div tag we expect here as it is better for customer to not spread info across the page but have it in place. I think that section bellow about DIV container should be moved here

Author: I don't think it should be moved here. This is kinda like a high level overview. I would rather move all the info down.

|hr|

.. _Talkable Container DIV:

Talkable Container DIV
~~~~~~~~~~~~~~~~~~~~~~

> Add the following Talkable Container DIV in the body of every page where you want to show a loyalty campaign:

Reviewer: ```suggestion
Add the following Talkable Container DIV in the body of every page where you want a loyalty campaign to be shown:
```

.. code-block:: html

   <div id="talkable-loyalty"></div>

> The `Talkable Initialization Script <Initialization Script_>`_ must be present in your head template in order for the Loyalty Dashboard to work. The placement of the DIV is important since it's going to determine where the campaign will be shown.

Reviewer: ```suggestion
The `Talkable Initialization Script <Initialization Script_>`_ must be present in your head template in order for the Loyalty Dashboard to work. The placement of the DIV container/tag is important since it's going to determine where the campaign will be shown.
```

.. note::

>    The same DIV is used for both Loyalty Dashboard and Loyalty Redeem Widget.

Reviewer: ```suggestion
   The same DIV container/tag is used for both Loyalty Dashboard and Loyalty Redeem Widget.
```

>    You can find more info about where to place the DIV in :ref:`Loyalty Dashboard Placement <campaigns/campaign_placements/loyalty_dashboard>` and :ref:`Loyalty Widget Placement <campaigns/campaign_placements/loyalty_widget>`.

Reviewer: ```suggestion
   You can find more info about where to place the DIV container/tag in :ref:`Loyalty Dashboard Placement <campaigns/campaign_placements/loyalty_dashboard>` and :ref:`Loyalty Widget Placement <campaigns/campaign_placements/loyalty_widget>`.
```

Loyalty Dashboard Notes
-----------------------

The dashboard can be added inline inside your user accounts menu, however the dashboard content width for proper display is 980px.

Loyalty Widget Notes
--------------------

> Add Talkable container DIV element at the checkout exactly where the loyalty points redemption widget should render.

Reviewer: ```suggestion
Add Talkable Container DIV at the checkout exactly where the loyalty points redemption widget should rendered.
```

> Add Talkable Container DIV at the checkout exactly where the loyalty points redemption widget should rendered.

Reviewer: And change please `If none provided` to `If none is provided` in `Loyalty Actions Notes`

Loyalty Dashboard
~~~~~~~~~~~~~~~~~

> Add the following Talkable Container DIV in the body of every page that has `Talkable Initialization Script <Initialization Script_>`_ included:

Author: > Also, I don't see notices on this page about email (authentication) to be required, maybe we should notice it on this page as well

I think it's better to link the placement pages here since they have a lot of relevant information.

path (`www.your-site.com/loyalty`), add the Talkable Container DIV in
the body of the page and show loyalty campaign with the JS API function:
Add the following Talkable Container DIV in the body of every page that has `Talkable Initialization Script <Initialization Script_>`_ included:

.. code-block:: html

   <div id="talkable-loyalty"></div>
   <script>
     window._talkableq.push(['show_loyalty', {

>        optin: true // optional, determines whether a customer agreed to participate in the loyalty program

Reviewer: This question is not related to your task, but is the consent handled with new placement integration approach?

Author: Looks like with the new approach there's no way to provide the `optin` option unless you call `show_loyalty` manually.

     }]);
   </script>

Loyalty Dashboard Notes
-----------------------

1. The `Talkable Initialization Script <Initialization Script_>`_ must be
   present in your head template in order for the Loyalty Dashboard to work.

2. Talkable will inject loyalty content where Talkable Container
   resides in your DOM. Adding a new page is only a suggestion. The
   dashboard can be added inline inside your user accounts menu, however
2. The dashboard can be added inline inside your user accounts menu, however
   the dashboard content width for proper display is 980px.

3. Allow your users to reach the Loyalty Dashboard by adding a link
   from any place that makes sense considering your website configuration.
   Most common uses are links in the user accounts section, or from the user accounts menu.

> 4. `show_loyalty` optionally accepts `email`, `first_name`, `last_name`, and `custom_properties`,

Reviewer: I'd mention `first_name`, `last_name` and `custom_properties` as a possible data can be passed in `Data Capture` section at the top of the document

   which override respective values from `authenticate_customer`. Email must be provided in either of the sources to
   show loyalty dashboard.

|hr|

.. _Loyalty Redeem Widget:

Loyalty Redeem Widget
~~~~~~~~~~~~~~~~~~~~~

Add Talkable container DIV element at the checkout

> exactly where the loyalty points redemption widget should render, and call JS API function to render the widget:

Reviewer: Thats what I'm talking about, current version of documentation misses

> Add Talkable container DIV element at the checkout exactly where the loyalty points redemption widget should render

clarification
